playbooks/templates/.env.j2

```diff
@@ -41,9 +34,15 @@ SERVER_DOMAIN={{ server_domain }}
 # A string representing name of your portal e.g. 'my-portal.com' or 'My Awesome
 # Skynet Portal' (internal use only)
 #
-# REQUIRED: Manually defined by user 
+# REQUIRED: Manually defined by user, or Ansible automatically generated 
 PORTAL_NAME={{ webportal_server_config.portal_name }}
 
+# Sia api password and server uid. 
+#
+# REQUIRED: manually defined by user, or Ansible automatically generated 
+SIA_API_PASSWORD={{ webportal_server_config.sia_api_password }}
+SERVER_UID={{ webportal_server_config.server_uid }}
```

Comment:
can we add a smol comment above this variable

Reply:
Added comment, please review.